addons/mail/models/mail_gateway_allowed.py

```diff
@@ -21,11 +20,33 @@ class MailGatewayAllowed(models.Model):
     _description = 'Mail Gateway Allowed'
     _name = 'mail.gateway.allowed'
 
-    email = fields.Char('Email')
+    email = fields.Char('Email Address')
     email_normalized = fields.Char(
         string='Normalized Email', compute='_compute_email_normalized', store=True, index=True)
 
     @api.depends('email')
     def _compute_email_normalized(self):
         for record in self:
             record.email_normalized = tools.email_normalize(record.email)
+
+    def action_list_view(self):
```

Comment:
It looks like there is a `get_empty_list_help` that seems to allow building dynamic helper messages such as this.

Maybe it would simplify everything?